src/citrine/informatics/experiment_values.py

```diff
@@ -0,0 +1,130 @@
+from typing import Type
+
+from citrine._serialization.serializable import Serializable
+from citrine._serialization.polymorphic_serializable import PolymorphicSerializable
+from citrine._serialization import properties
+
+__all__ = [
+    'ExperimentValue',
+    'RealExperimentValue',
+    'IntegerExperimentValue',
+    'CategoricalExperimentValue',
+    'MixtureExperimentValue',
+    'ChemicalFormulaExperimentValue',
+    'MolecularStructureExperimentValue'
+]
+
+
+class ExperimentValue(PolymorphicSerializable['ExperimentValue']):
+    """An container for experiment values.
+
+    Abstract type that returns the proper type given a serialized dict.
+    """
+
+    @classmethod
+    def get_type(cls, data) -> Type[Serializable]:
+        """Return the subtype."""
+        return {
+            "RealValue": RealExperimentValue,
+            "IntegerValue": IntegerExperimentValue,
+            "CategoricalValue": CategoricalExperimentValue,
+            "MixtureValue": MixtureExperimentValue,
+            "OrganicValue": ChemicalFormulaExperimentValue,
+            "InorganicValue": MolecularStructureExperimentValue,
+        }[data["type"]]
```

Comment:
`typ` elsewhere is in snake_case.  Have we introduced inconsistency into the exchange?

Reply:
The only place I've seen snake-cased type discriminators is data source references ([data_sources.py](https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/informatics/data_sources.py)). All the others I'm aware of ([predictors](https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/informatics/predictors/predictor.py), [dimensions](https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/informatics/dimensions.py), [design spaces](https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/informatics/design_spaces/design_space.py), etc) use camel case.

Reply:
All of the artifacts on the data side of the platform are in snake_case.
https://github.com/CitrineInformatics/citrine-python/blob/06bd2cb72b4ab04b7ecc9fe258ed4ac3f0a46635/src/citrine/informatics/data_sources.py#L68-L72
Mixed behavior:
https://github.com/CitrineInformatics/citrine-python/blob/06bd2cb72b4ab04b7ecc9fe258ed4ac3f0a46635/src/citrine/informatics/predictors/predictor.py#L24-L27
DataConcepts.py:
https://github.com/CitrineInformatics/citrine-python/blob/06bd2cb72b4ab04b7ecc9fe258ed4ac3f0a46635/src/citrine/resources/data_concepts.py#L74-L77
gemd-python:
https://github.com/CitrineInformatics/gemd-python/blob/1b6c1866aa5844b5a748a8f89ac64535d2776075/gemd/entity/object/material_run.py#L53
Team.py:
https://github.com/CitrineInformatics/citrine-python/blob/06bd2cb72b4ab04b7ecc9fe258ed4ac3f0a46635/src/citrine/resources/team.py#L141-L144

Reply:
Also, all the `typ`s from GemTables:
https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/gemtables/rows.py
https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/gemtables/variables.py
https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/gemtables/columns.py

Reply:
Fair enough. But that's neither here nor there with regards to citrine-python and this PR.